Traceback (most recent call last):
  File "C:\Users\colli\Documents\Python\Platformer Game\code\main.py", line 33, in <module>
    main()
  File "C:\Users\colli\Documents\Python\Platformer Game\code\main.py", line 18, in main
    g_engine.run()
  File "C:\Users\colli\Documents\Python\Platformer Game\code\engine\engine.py", line 131, in run
    obj.update()
  File "C:\Users\colli\Documents\Python\Platformer Game\code\player.py", line 68, in update
    self.update_abilities()
  File "C:\Users\colli\Documents\Python\Platformer Game\code\_player_abilities.py", line 36, in update_abilities
    self.ability_rope( self.has_ability( 'rope' ) )
  File "C:\Users\colli\Documents\Python\Platformer Game\code\_player_abilities.py", line 142, in ability_rope
    self._rope_unhook()
  File "C:\Users\colli\Documents\Python\Platformer Game\code\_player_abilities.py", line 245, in _rope_unhook
    self.engine.play_sound( f'rope_release_p{ pitch_index }' )
  File "C:\Users\colli\Documents\Python\Platformer Game\code\engine\_engine_mixer.py", line 49, in play_sound
    raise KeyError( f'Unknown sound "{ name }"' )
KeyError: 'Unknown sound "rope_release_p5"'
